README.md

```diff
@@ -38,34 +38,70 @@ The key/value pairs insides this ConfigMap are:
 
 #### Pointer to binding data
 
-This specification supports different scenarios for exposing bindable data.  
+This specification supports different scenarios for exposing bindable data. Below is a summary of [how to indicate what is interesting for binding](annotations.md).
 
 1. OLM-enabled Operator: Use the `statusDescriptor` and/or `specDescriptor` parts of the CSV to mark which `status` and/or `spec` properties reference the [binding data](#service-binding-schema):
     * The reference's `x-descriptors` with a possible combination of:
       * ConfigMap:
-        * `servicebinding:configMap`
+        
+            - path: data.dbcredentials
+              x-descriptors:
+                - urn:alm:descriptor:io.kubernetes:ConfigMap 
+                - servicebinding
+
       * Secret:
-        * `servicebinding:secret`
-      * Individual binding items:
-        * `servicebinding:secret:host`
-        * `servicebinding:secret:port`
-        * `servicebinding:secret:endpoints`
-        * `servicebinding:secret:uri`
-        * `servicebinding:secret:<binding_property>`  (where `<binding_property>` is any property from the binding schema)
 
-2. Non-OLM Operator: - An annotation in the Operator's CRD to mark which `status` properties reference the [binding data](#service-binding-schema).  The value of this annotation can be specified in either [JSONPath](https://kubernetes.io/docs/reference/kubectl/jsonpath/) or [GO templates](https://golang.org/pkg/text/template/):
+            - path: data.dbcredentials
+              x-descriptors:
+                - urn:alm:descriptor:io.kubernetes:Secret 
+                - servicebinding
+
+
+      * Individual binding items:
+        * ```
+            - urn:alm:descriptor:io.kubernetes:Secret 
```

Comment:
should we clarify that if they are placing this descriptor under a single path (i.e. `path: status.host`), then `urn:alm:descriptor:io.kubernetes:Secret` would not apply?

Reply:
Good point, I've updated the PR with annotations/descriptors for this.